auto_round/utils.py

```diff
@@ -1597,6 +1614,30 @@ def mllm_load_model(
     return model, processor, tokenizer, image_processor
 
 
+def diffusion_load_model(
+    pretrained_model_name_or_path,
+    device="cpu",
+    torch_dtype="auto",
+    use_auto_mapping=True,
```

Comment:
better set the use_auto_mapping to False, we only enable it when users passed multiple devices

Reply:
updated